Update README.md (#168)
Vita-App/Vita: README.md

<!-- ALL-CONTRIBUTORS-LIST:END -->

This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
<h2 align="center">Code of Conduct</h2>

To maintain a safe and inclusive space for everyone to learn and grow, contributors are advised to follow the [Code of Conduct](./CODE_OF_CONDUCT.md). 
 

<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->

<h2 align="center">Feedback</h2>

> If you have any feedback or suggestions please reach out to Project - Maintainer.  
> * [Lalit Kumar Singh](https://github.com/Lalit3716) 
> 
> 
> Or you can create a  <a href="https://github.com/Vita-App/Vita/issues">issue</a> and mention there , which new features or extension can make this Project more good.

Reviewer: ```suggestion
If you have any feedback or suggestions you can create an  <a href="https://github.com/Vita-App/Vita/issues">issue</a> and mention there , which new features or extension can make this Project more good.
```

<!-- ALL-CONTRIBUTORS-LIST:END -->

This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
<h2 align="center">Code of Conduct</h2>

To maintain a safe and inclusive space for everyone to learn and grow, contributors are advised to follow the [Code of Conduct](./CODE_OF_CONDUCT.md). 
 

<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->

<h2 align="center">Feedback</h2>

> If you have any feedback or suggestions please reach out to Project - Maintainer.  
> * [Lalit Kumar Singh](https://github.com/harshita214) 

Reviewer: ```suggestion
If you have any feedback or suggestions please reach out to: 
* [Lalit Kumar Singh](https://github.com/Lalit3716) 
```

> Or you can create a  <a href="https://github.com/aman34503/harshita214/issues">issue</a> and mention there , which new features or extension can make this Project more good.

Reviewer: Correct the url in `href` here

<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->

<br>
  

<hr>

<div align="center">

### Show some ❤️ by starring this awesome Repository!

</div>

<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
<hr>

<h3 align="center"> 

 ‼️ For [JWOC'23](https://jwoc.tech/) program Admin and Mentor rights of Vita App are given to [Lalit Kumar Singh](https://github.com/whiteHatpro) ‼️

</h3>
  
  
<div id="Bottom"></div>

> 

Reviewer: Remove this section

<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->

<h2 align="center">Feedback</h2>

If you have any feedback or suggestions please reach out to Project - Maintainer.  
* [Lalit Kumar Singh](https://github.com/Lalit3716) 

Or you can create a  <a href="https://github.com/Vita-App/Vita/issues">issue</a> and mention there , which new features or extension can make this Project more good.

<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->

<br>

>   

Reviewer: Don't remove this section just make some changes I told you to make in previous review
